### PR TITLE
Fix fallback demo app template code

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Fall back component is not interactive so changing the demo to reflect that.

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

**Comment by abidlabs** [comment]
Sorry I don't have enough context to review this, what is the Fallback component?

**Comment by freddyaboulton** [comment]
My bad! What you get when you don't choose `--template` in the `create` command

File: test/test_gradio_component_cli.py [hunk 1]
```diff
@@ -121,3 +121,30 @@ def test_install(tmp_path):
...
+    app = (tmp_path / "demo" / "app.py").read_text()
+    answer = textwrap.dedent(
+        """
+
+import gradio as gr
+from gradio_simplecomponent2 import SimpleComponent2
+
+
+with gr.Blocks() as demo:
+    gr.Markdown("# Change the value (keep it JSON) and the the front-end update automatically.")
```
Comment (abidlabs) [inline test/test_gradio_component_cli.py:142]:
```suggestion
    gr.Markdown("# Change the value (keep it JSON) and the front-end will update automatically.")
```

File: gradio/cli/commands/components/_create_utils.py [hunk 2]
```diff
@@ -44,6 +44,12 @@ def _in_test_dir():
         gr.Number(value=10, interactive=True)
 """
 
+fallback_code = """
+with gr.Blocks() as demo:
+    gr.Markdown("# Change the value (keep it JSON) and the the front-end update automatically.")
```
Comment (abidlabs) [inline gradio/cli/commands/components/_create_utils.py:49]:
```suggestion
    gr.Markdown("# Change the value (keep it JSON) and the front-end will update automatically.")
```

**Comment by abidlabs** [comment]
Awesome, thanks @freddyaboulton!

**Review by abidlabs — APPROVED** [review]

**Pull request merged**